.github/workflows/wheels.yaml

```diff
@@ -23,44 +23,20 @@ jobs:
       fail-fast: false
 
     steps:
-      - uses: actions/setup-python@v2
-        name: Install Python
-        with:
-          python-version: 3.9
-
-      - uses: s-weigand/setup-conda@v1
-        if: matrix.os == 'windows-latest'
-        with:
-          update-conda: true
-          conda-channels: conda-forge
-          activate-conda: true
-          python-version: 3.9
-
       - uses: actions/checkout@v2
-      - name: Install cibuildwheel
-        run: python -m pip install cibuildwheel==2.2.2
-
-      - name: Install dependencies and yt
-        shell: bash
-        env:
-          dependencies: "full"
-          LDFLAGS: "-static-libstdc++"
-        run: source ./tests/ci_install.sh
 
       - name: Build wheels for CPython
-        run: |
-          python -m cibuildwheel --output-dir dist
+        uses: pypa/cibuildwheel@v2.3.0
         env:
           CIBW_BUILD: "cp37-* cp38-* cp39-* cp310-*"
-          CIBW_SKIP: "*-musllinux_*"  # these fail due to side effects from previous builds
```

Comment:
Just as I suspected, prolonged build times are caused by numpy build. Since there are no musl wheels available for numpy, I believe we should keep this `CIBW_SKIP`. Let's not be overeager and don't build all the things...

Reply:
agreed ! thanks for catching this